terraform/modules/kafka/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_cloudwatch_log_group" "msk_broker_logs" {
```

Comment:
I think this should not be part of the module as perhaps we would not like to store logs in cloudwatch, this should be part of main.tf and pass the resource name as var - if applied -

Reply:
This will not be a public reusable module. We could define the logs to be dynamically set but this won't provide us with any benefit in this project just make the kafka module more complex